File: Services/Student/Student.Core/Entities/Course.cs
namespace Student.Core.Entities;

[System.ComponentModel.DataAnnotations.Schema.Table("course", Schema = "std")]
public class Course
{
    [Dapper.Contrib.Extensions.ExplicitKey, System.ComponentModel.DataAnnotations.Required]
    [System.ComponentModel.DataAnnotations.Schema.Column("id")]
    public string Id { get; set; }
    [System.ComponentModel.DataAnnotations.Schema.Column("name")]
    public string Name { get; set; }
    [System.ComponentModel.DataAnnotations.Schema.Column("description")]
    public string Description { get; set; }

}

File: Services/Teacher/Teacher.Core/Entities/Course.vb
<System.ComponentModel.DataAnnotations.Schema.Table("course", Schema := "std")>
Public Class Course
    <Dapper.Contrib.Extensions.ExplicitKey, System.ComponentModel.DataAnnotations.Required>
    <System.ComponentModel.DataAnnotations.Schema.Column("id")>
    Public Property Id As String

    <System.ComponentModel.DataAnnotations.Schema.Column("name")>
    Public Property Name As String

    <System.ComponentModel.DataAnnotations.Schema.Column("description")>
    Public Property Description As String
End Class